src/Core/Framework/Plugin/KernelPluginLoader/KernelPluginLoader.php

```diff
@@ -73,26 +73,37 @@ public function __construct(ClassLoader $classLoader, ?string $pluginDir = null)
         return $this->pluginInstances;
     }
 
-    final public function getBundles($kernelParameters = []): iterable
+    final public function getBundles($kernelParameters = [], array $skipBundleNames = []): iterable
```

Comment:
please rename `skipBundleNames` to  `loadedBundles`

Reply:
But this is the outer experience that this is about the already loaded bundles. Inside it is just about to skip these as well. Loaded bundles therefore depends on the caller.

Reply:
I am open to rename it but I want to understand your thoughts on it.

Reply:
You use the bundles to be skipped internally, but the external function does not know that. From the semantic point of view it's nicer here in my opinion:

"give me bundles, here are the already loaded"

You also enrich this array internally, with the bundles you loaded.

Reply:
Good point. Gotta change that.